### PR TITLE
Set reporting period for all meters

### PR DESCRIPTION
**Pull request by jjzeng-seattle**
In addition, set it for all newly-created meters.

### CODE REVIEW

File: metrics/config.go [hunk 1]
```diff
@@ -296,6 +296,8 @@ func createMetricsConfig(ops ExporterOptions, logger *zap.SugaredLogger) (*metri
 		mc.reportingPeriod = 60 * time.Second
 	} else if mc.backendDestination == prometheus {
 		mc.reportingPeriod = 5 * time.Second
+	} else if mc.backendDestination == openCensus {
+		mc.reportingPeriod = 60 * time.Second
 	}
```
Comment (yanweiguo) [inline metrics/config.go:301]: Change this `else if` section to a `switch`?

File: metrics/resource_view.go [hunk 2]
```diff
@@ -186,6 +199,11 @@ func meterExporterForResource(r *resource.Resource) *meterExporter {
 	mE.m = view.NewMeter()
 	mE.m.SetResource(r)
 	mE.m.Start()
+
+	mc := getCurMetricsConfig()
+	if mc != nil {
+		mE.m.SetReportingPeriod(mc.reportingPeriod)
```
Comment (yanweiguo) [inline metrics/resource_view.go:205]: Is it possible to add an unit test for it?

File: metrics/config_test.go [hunk 3]
```diff
@@ -281,6 +281,7 @@ var (
 			domain:             servingDomain,
 			component:          testComponent,
 			backendDestination: openCensus,
+			reportingPeriod:    60 * time.Second,
```
Comment (vagababov) [inline metrics/config_test.go:284]:
```suggestion
			reportingPeriod:    time.Minute,
```

File: metrics/config.go (hunk 1 above)
Comment (jjzeng-seattle) [reply metrics/config.go]: done.

File: metrics/resource_view.go (hunk 2 above)
Comment (jjzeng-seattle) [reply metrics/resource_view.go:205]: I thought about it.  It's hard to verify the internal of a meter.  Testing the effect(waiting for the reports) is not suitable for unit tests.

File: metrics/config_test.go (hunk 3 above)
Comment (jjzeng-seattle) [reply metrics/config_test.go:284]: done.

Comment (yanweiguo) [reply metrics/config_test.go:284]: no change here.

**Pull request merged**